template.tpl

```diff
@@ -363,6 +368,54 @@ if (queryPermission('inject_script', url)) {
   log('🧪 Bambuser Debugger 🧪\n', '🚫 Script load failed due to permissions mismatch. You may need to whitelist your custom script URL ('+ url +') in the template setting > permissions > Allowed URL match patterns.');
   data.gtmOnFailure();
 }
+} else if (data.feature === 'oneToOneIntegration') {
+  const conf = {
```

Comment:
In this config I think we should for now add the default
allowFirstPartyCookies: false
until we have adapted to consent API